### DISCUSSION TITLE
Visual Thinking Strategies: a rigorous theory of aesthetic development with strong empirical grounding

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#457] [Research]
https://vtshome.org

Close connection to ego-development in general.

https://vtshome.org/research/

> Coined by German-born author and psychologist Rudolf Arnheim, whose primary book shares the same name, “Visual Thinking” stands paramount in Abigail Housen’s “empirical research” and resulting theory of aesthetic development. The application of Housen, Arnheim, Piaget and others constitute the genesis and ongoing theoretical underpinnings behind the development of Visual Thinking Strategies methods and curricula.
>
> Through the process of collecting and analyzing Aesthetic Development Interviews (see Research Methods) and identifying and understanding the concrete words and ideas of novice viewers in the moment and over time, Housen derived the developmental stage theory that informs all VTS methods and curricula. In Housen’s essay [Eye of the Beholder: Research, Theory and Practice](https://vtshome.org/wp-content/uploads/2016/08/Eye-of-the-Beholder.pdf), she provides an account of how she began her study of Aesthetic Development, how she came up with her research methodology, and the overall findings of her 20-years-old research. Housen also briefly discusses how she applied what she learned to creating the Visual Thinking Strategies curriculum.

**Comment by baouroux** [comment]
Absolutely fascinating, thanks for the post. I know of an Italian consulting Organisational Design company (https://www.peoplerise.net/en/) running faciliation sessions in art galleries, and I am scheduled to present at a conference of theirs in June.
There is a lot of resonance with a theme I am developing of 'signifiers' in the actual experience of creativity: https://www.tyler.world/complexity
...And how this relates to navigating complexity. Even Dave Snowden I hear refering to the 'evolutionary purpose of art'!
